### PR TITLE
Integrate final rejection into pipeline 

### PR DESCRIPTION
**Pull request by F-said**

### CODE REVIEW

**Comment by georgebuzzell** [comment]
@F-said did you notice the failed checks?

File: base_eeg/test.py [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 import pathlib
-import json
-import autoreject as ar
-from find_montage import get_montage
 from mne_bids import BIDSPath, read_raw_bids
 import finalRej as fr
-    
+
+
 def main():
```
Comment (SDOsmany) [inline base_eeg/test.py:7]: i think the linter is complaining about only leaving 1 space here , I think you you put the comment one line lower it the complaint will go away. documentation [here](https://www.flake8rules.com/rules/E305.html)

**Review by SDOsmany — COMMENTED** [review]
besides the linting check. Everything is great. I notice you haven't changed anything from the last PR, it's weird I don't remember the linter complaining

**Review by SDOsmany — APPROVED** [review]
LGTM!

**Comment by georgebuzzell** [comment]
Thank you, both!

**Pull request merged**